webapp/public/js/domjudge.js

```diff
@@ -361,52 +361,52 @@ function initFavouriteTeams()
 // Note that team,prob,submission IDs are as expected by iCAT.
 function postVerifyCommentToICAT(url, user, teamid, probid, submissionid)
 {
-	var msg = document.getElementById("comment");
-	if (msg === null) {
-		return;
-	}
+    var msg = document.getElementById("comment");
+    if (msg === null) {
+        return;
+    }
 
-	var form = document.createElement("form");
-	form.setAttribute("method", "post");
-	form.setAttribute("action", url);
-	form.setAttribute("hidden", true);
+    var form = document.createElement("form");
+    form.setAttribute("method", "post");
+    form.setAttribute("action", url);
+    form.setAttribute("hidden", true);
 
-	// setting form target to a window named 'formresult'
-	form.setAttribute("target", "formresult");
+    // setting form target to a window named 'formresult'
+    form.setAttribute("target", "formresult");
 
-	function addField(name, value) {
-		var field = document.createElement("input");
-		field.setAttribute("name", name);
-		field.setAttribute("id",   name);
-		field.setAttribute("value", value);
-		form.appendChild(field);
-	};
+    function addField(name, value) {
+        var field = document.createElement("input");
+        field.setAttribute("name", name);
+        field.setAttribute("id",   name);
+        field.setAttribute("value", value);
+        form.appendChild(field);
+    };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
	}
```